dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/TicketCacheBindTest.java

```diff
@@ -57,6 +65,25 @@ public void basicLoginChecks() throws Exception {
         baselineTests();
     }
 
+    /**
+     * Run golden path tests for bindAuthMech=GSSAPI, a valid krbPrincipalName and valid krb5TicketCache
+     *
+     * @throws Exception
+     */
+    @Test
+    @CheckForLeakedPasswords(LdapKerberosUtils.BIND_PASSWORD)
+    @AllowedFFDC("javax.naming.NamingException") // temporary, remove when Issue #16231 is fixed
```

Comment:
Do you have a comment in 16231 to come back and remove this?

Reply:
It does-- #16231